journal-de-la-societe-des-oceanistes.csl

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- Polyglot; journal publishes in English and French. -->
```

Comment:
@POBrien333, can you confirm the default-locale should be British English, and that this is not either a French or polyglot journal?

Reply:
as usual I forgot to take the locale out. thanks for spotting.